Design.md

```diff
@@ -0,0 +1,96 @@
+
+LZW or another compression method will be used for all files to reduce file sizes.
+
+## Cumulative Displacement
+### Format
+To fully understand the context of InSAR-derived displacement data (such as the OPERA-Disp products) multiple pieces are required for each pixel:
+
+1. The geographic location of the pixel (from the GDAL metadata)
+2. The start and end date for the displacement observation
+3. The reference pixel location to which all displacement measurements in a product are relative to
+
+This is a lot of information, and we will need to ensure that we provide access to all of this data (or at least make it trackable) for every pixel in the mosaic.
+
+To do this, the 1x1 degree cumulative displacement COGs will have the following components:
+1. A Float32 short wavelength cumulative displacement band sourced directly from the products
+2. A sidecar metadata file (likely a STAC item) containing frame, spatiotemporal reference data, and a link to an external tile frame image (see below)
+
+The sidecar metadata will contain the following fields:
+1. List of OPERA frames present in the cumulative displacement COG
+2. The frame reference and secondary date in the format YYYY/MM/DD for each frame present in the tile
+3. The frame reference point location encoded as both the array coordinates (row/column) and the geographic coordinates of the reference pixel in the native projection of the frame for each frame present in the tile
+4. A URL pointing to the location of OPERA frame image for that tile (see below)
+
+For example, the fields in metadata raster may look like:
+```
+OPERA_FRAMES: "1, 2"
+FRAME_1_REF_POINT: "129877.23, 128383.28"
+FRAME_1_EPSG: "12345"
+FRAME_1_REF_POINT_ARRAY: "10, 20"
+FRAME_1_REF_DATE: "2015/01/01"
+FRAME_1_SEC_DATE: "2015/01/02"
+FRAME_2_REF_POINT: "22348.23, 38973.28"
+FRAME_2_EPSG: "12345"
+FRAME_2_REF_POINT_ARRAY: "20, 11"
+FRAME_2_REF_DATE: "2015/02/01"
+FRAME_2_SEC_DATE: "2015/02/02"
+```
+
+In addition to this per-COG information, there will also be one accompanying metadata raster (in COG format) for each tile location (i.e., one per footprint). The metadata raster will have the same resolution, extent, and projection as the cumulative displacement COGs, and in fact will serve as the template for create cumulative displacement COGs. The metadata raster will have the following components
+1. A UInt16 band indicating the frame number the cumulative displacement value each pixel is sourced from
+2. A sidecar metadata file with the same information as the cumulative displacement sidecar files, except for secondary date and OPERA frame image URL
```

Comment:
Reference date, secondary date, and reference point are all specific to a particular disp .nc product, and might all vary from product to product from the same frame, yes? What's the intent of capturing a reference point and reference date for the frame? Are we using this to document the fixed references all 1x1 COGs for that frame will be re-referenced to when we generate them?